src/components/pages/login/LoginForm.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react'
+
+export default function LoginForm() {
+  // state
+  const [login, setLogin] = useState("");
```

Comment:
```suggestion
  const [username, setUsername] = useState("");
```
"login" tout seul pourrait faire penser à un objet, ex: 

const login = {
    username : "david",
    password : "monMotDePasse"
}